### DISCUSSION TITLE
Extracting Audio Files?

### DISCUSSION THREAD

**Discussion by TheRenegadist** [Jones3D-The-Infernal-Engine/Documentation#1] [Q&A]
Hey, was quite surprised to stumble on this and the ProjectMarduck Github, I had no idea people were actually working on tools to mod this game.

Anyhow, I'm attempting to extract both the sound files found in the game as well as all of the dialogue for the various characters especially Indy, how would I go about doing this or is there an archive somewhere containing all of this?

Thanks in advance, hope people still check this page.

**Comment by smlu** [comment]
Ref: https://github.com/smlu/ProjectMarduk/issues/4#issuecomment-1085369160
